### ISSUE TITLE
Does not define interaction with 0-RTT

### ISSUE THREAD

**Issue by ctz** [tlswg/tls-flags#16]
RFC8446 4.2.10 requires that:

```
   Future extensions MUST define their interaction with 0-RTT.
```

This extension does not (as far as I can see) do that.

I think the text in tls-flags should prevaricate on this topic, because really it's an encoding framework for trivial extensions (that themselves need to define their interaction with 0-RTT.)

**Comment by yoavnir** [comment]
Created a PR.  Let me know if this is satisfactory.

https://github.com/tlswg/tls-flags/pull/17

**Comment by seanturner** [comment]
@yoavnir @ctz assume this one can be closed based on #17 being merged?
